innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+* InnerSource Portal
+
+## Patlet
+
+InnerSource projects at your company are increasing but potential contributors have no easy way of discovering them.
+
+Create an intranet website that indexes all available InnerSource projects.  Make it easy for InnerSource projects owners to include information about their projects and for visitors to the site to search across all the available projects.  This will enable potential contributors to more easily locate projects of interests and learn more about who is running them, the project goals and access technical and community management related documentation.
+
+## Problem
+
+There is no central, easily accessible resource for InnerSource project owners and potential contributors to leverage to advertise or discover InnerSource projects.  
+
+## Story
+
+You are attempting to establish an InnerSource practice within your organization.  You are aware of some projects being run using an InnerSource model, but their existence is only being communicated via word of mouth, email or sidebar conversations with other employees.  There is no single, shared resource for employees across the organization to access which will allow them to easily discover all ongoing InnerSource projects.  This is severely limiting the growth potential of every InnerSource project.  What can be done to help all InnerSource projects raise their visibility to as large an audience as possible and attract contributors organization-wide?
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+
+## Context
+
+* Managers have given tacit acceptance that their employees should take part in InnerSource projects
+* Employees desire the ability to be made aware of all of the InnerSource project activity occurring across the compnany
+* Owners of InnerSource projects desire the ability to attract as many potential contributors to their projects as possible
```

Comment:
Putting on the community leader hat now: I'd rather have a small but very competent team rather than a large team of newbies.

Reply:
Agreed.  However, the premise is that the Gig Marketplace would be most attractive to project owners who are attempting to establish even that first small core community of developers and less to those that already being run by a team of familiar trusted contributors.

Given that though, perhaps there should be some mention of that somewhere in the pattern.  That the Gigs posted to the marketplace can be used to attract an initial audience.